sssd_test_framework/utils/sssd.py

```diff
@@ -113,6 +113,7 @@ def setup(self) -> None:
     def async_start(
         self,
         service="sssd",
+        service_user="sssd",
```

Comment:
It feels weird that `service` can be configured to anything else but 'sssd' in 'SSSD' class.

Reply:
With systemd, we have many services. Most of them are controlled by sssd.service, but e.g. sssd-kcm is not. The original intention was to allow to also control sssd-kcm. However, Alejandor added commit to start/stop sssd-kcm with sssd as well so maybe it is not needed anymore.